### ISSUE TITLE
RedisConfig uses default value

### ISSUE THREAD

**Issue by devzorg** [jxskiss/ssl-cert-server#11]
```openresty-ssl-server-1  | 2022-09-15T11:26:55.775Z      ERROR   server  server/server.go:94     failed get certificate, typ= ACME_ON_DEMAND, name= *******.info {"error": "dial tcp 127.0.0.1:6379: connect: connection refused"}```

but it defined as
```
storage:
  type: "redis"  # or redis
  dir_cache: "/app/storage"
  redis:
    addr: "redis:6379"
    prefix: ""
```
please also add DB in struct to make possible choose db index, and may some more options like password and etc able in default lib of redis

**Comment by jxskiss** [comment]
Good point

**Comment by jxskiss** [comment]
Hi @devzorg, I have checked the doc of go-redis, the configuration already supports chosing db and other options, you can specify it as `redis://<user>:<password>@<host>:<port>/<db_number>`, you may check <https://pkg.go.dev/github.com/go-redis/redis/v8#ParseURL> for details of the format.

**Comment by jxskiss** [comment]
Though, this is a good issue, I will update the example conf to make it a little clear of the format.

**Comment by devzorg** [comment]
> Hi @devzorg, I have checked the doc of go-redis, the configuration already supports chosing db and other options, you can specify it as `redis://<user>:<password>@<host>:<port>/<db_number>`, you may check https://pkg.go.dev/github.com/go-redis/redis/v8#ParseURL for details of the format.

you mean using the 'addr' key of yml config  ?

**Comment by jxskiss** [comment]
Yes, use `redis://<user>:<password>@<host>:<port>/<db_number>` as value of addr to specify options to connect to Redis.

**Comment by jxskiss** [comment]
Fixed by #12, goingq to close this.
